includes/sdk-debuggability/browser.md

```diff
@@ -0,0 +1,68 @@
+#### How to debug
+
+Debugging in a browser can help you identify problems related to your code's implementation, as well as potential issues within the SDKs you're using. Here's a basic guide on how to use the browser's built-in Developer Tools (DevTools) for debugging.
+
+##### Console
+
+You can find JavaScript errors under **Inspect > Console**, which might have the details about the line of code and file that caused the problem. The console also allows you to execute JavaScript code in real-time.
+
+* Enable debug mode by following these [instructions](./#debug-mode). Then With the default logger, extra function context information will be output to the developer console when any SDK public method is invoked, which can be helpful for debugging.
+
+* If you're able to send the event successfully after entering `amplitude.init(API_KEY, 'USER_ID')` in the browser console, it indicates that your `amplitude.init` call might not have been triggered. We support deferred initialization, so the events will be dispatched after the initialization call. Therefore, please check your implementation.
+
+##### Network Request
```

Comment:
Can we add a link here to [HTTP V2 response format](https://www.docs.developers.amplitude.com/analytics/apis/http-v2-api/#response-format)? and call out that some 400 error can be caused by [invalid user/device id ](https://www.docs.developers.amplitude.com/analytics/apis/http-v2-api/#device-ids-and-user-ids-minimum-length)

Reply:
This has been mentioned in the general debug doc at the ` vent Dropping
` section. Don't want to duplicate the words.